smalld/smalld.py

```diff
@@ -89,6 +95,29 @@ def run(self):
             time.sleep(5)
 
 
+class Intent(Flag):
+    # Gateway intents https://discord.com/developers/docs/topics/gateway#gateway-intents
```

Comment:
Style nit: I've tended to favor leaving out links like this, because they could change. Also because smalld doesn't aim to hide the api, it's reasonable to expect someone to look up stuff in the discord api docs when using smalld.